.pipelines/templates/assign-user-identity.yaml

```diff
@@ -5,7 +5,15 @@ parameters:
 
 steps:
   - script: |
-      VMSS_NAMES=$(az vmss list -g ${CLUSTER_RESOURCE_GROUP} --query "[].name" -o tsv)
+      for i in $(seq 1 180); do
```

Comment:
Adding these retries because vmss list returns empty in some CI runs after the AKS cluster is deployed: https://dev.azure.com/azure/secrets-store-csi-driver-provider-azure/_build/results?buildId=19509&view=logs&j=7f33e5bd-7764-5d8a-ba2e-506e078b9c3f&t=6967100b-a467-593c-8b42-ddff9425a044&s=d654deb9-056d-50a2-1717-90c08683d50a

Reply:
Seeing few errors like this in this run. can you pls confirm this is expected?

```console
2021-02-10T00:11:39.2230615Z E0210 00:08:17.324761       1 reconciler.go:488] "failed to reconcile spc for pod" err="failed to rotate objects for pod certificates-td28f/nginx-secrets-store-inline-crd-certs, err: rpc error: code = Unknown desc = failed to mount objects, error: failed to write file pkcs12cert1-secret-pfx at /var/lib/kubelet/pods/2f76e8d4-9f34-418f-b407-921506674686/volumes/kubernetes.io~csi/secrets-store-inline/mount: open /var/lib/kubelet/pods/2f76e8d4-9f34-418f-b407-921506674686/volumes/kubernetes.io~csi/secrets-store-inline/mount/pkcs12cert1-secret-pfx: no such file or directory" spc="azure-certs" pod="nginx-secrets-store-inline-crd-certs" controller="rotation"
```

Reply:
This happens if the rotation reconciler has begun and at the same time the pod is deleted. The target path no longer exists. The line just above that log indicates that:
```bash
2021-02-10T00:11:39.2227440Z I0210 00:08:17.310984       1 nodeserver.go:252] "node unpublish volume complete" targetPath="/var/lib/kubelet/pods/2f76e8d4-9f34-418f-b407-921506674686/volumes/kubernetes.io~csi/secrets-store-inline/mount"
2021-02-10T00:11:39.2230615Z E0210 00:08:17.324761       1 reconciler.go:488] "failed to reconcile spc for pod" err="failed to rotate objects for pod certificates-td28f/nginx-secrets-store-inline-crd-certs, err: rpc error: code = Unknown desc = failed to mount objects, error: failed to write file pkcs12cert1-secret-pfx at /var/lib/kubelet/pods/2f76e8d4-9f34-418f-b407-921506674686/volumes/kubernetes.io~csi/secrets-store-inline/mount: open /var/lib/kubelet/pods/2f76e8d4-9f34-418f-b407-921506674686/volumes/kubernetes.io~csi/secrets-store-inline/mount/pkcs12cert1-secret-pfx: no such file or directory" spc="azure-certs" pod="nginx-secrets-store-inline-crd-certs" controller="rotation"

```